src/app/pages/dashboard/components/test-results-search-card/test-results-search-card.html

```diff
@@ -1,12 +1,12 @@
 <button clear (click)="navigateToTestResultsSearch()" test-id="search-for-completed-tests">
-  <ion-card class="search-card">
+  <ion-card class="search-card" id="completed-search-card">
     <div id="card-content-wrapper">
       <ion-grid>
         <ion-row class="icon-row">
           <img src="assets/imgs/dashboard/search-tests.png" alt="Search tests" class="icon-search-tests"/>
         </ion-row>
         <ion-row>
-          <ion-button class="mes-neutral-button">
+          <ion-button id="completed-search" class="mes-neutral-button">
```

Comment:
same again, call this "completed-search-button"

Reply:
done